### PR TITLE
Feature #4

### LINKED ISSUE #4: Set Libraries:
- [x] tailwind
- [x] uuid
- [x] dotenv
- [x] TypeBox/ZOD
- [x] Nodemailer
- [x] date-fns
- [x] leaflet
- [x] Open Street Maps
- [x] Zustand
- [x] Mapbox
- [x] Openlayers
- [x] Axios

### PR DESCRIPTION
**Pull request by inpatr**
closes #4

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
 html,
```
Comment (dnmct) [inline styles/globals.css]: remove previous styles

File: tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  content: [
+    "./pages/**/*.{js,ts,jsx,tsx}",
+    "./components/**/*.{js,ts,jsx,tsx}",
+  ],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+};
```
Comment (dnmct) [inline tailwind.config.js:21]: override provided color scheme with own color scheme

**Comment by dnmct** [comment]
merge dev into your branch and resolve conflicts

**Review by dnmct — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 2 above)
Comment (inpatr) [reply tailwind.config.js:21]: closes #20 ?

**Comment by dnmct** [comment]
resolve conflicts again pls

File: tailwind.config.js (hunk 2 above)
Comment (dnmct) [reply tailwind.config.js:21]: yes

**Review by dnmct — APPROVED** [review]

**Pull request merged**